crates/engine/tree/src/tree/mod.rs

```diff
@@ -1738,15 +1763,15 @@ where
     /// Returns an error if we failed to fetch the state from the database.
     fn state_provider(&self, hash: B256) -> ProviderResult<Option<StateProviderBox>> {
         if let Some((historical, blocks)) = self.state.tree_state.blocks_by_hash(hash) {
-            debug!(target: "engine::tree", %hash, %historical, "found canonical state for block in memory");
+            debug!(target: "engine::tree", ?hash, ?historical, "found canonical state for block in memory");
             // the block leads back to the canonical chain
             let historical = self.provider.state_by_block_hash(historical)?;
             return Ok(Some(Box::new(MemoryOverlayStateProvider::new(historical, blocks))))
         }
 
         // the hash could belong to an unknown block or a persisted block
         if let Some(header) = self.provider.header(&hash)? {
-            debug!(target: "engine::tree", %hash, number = %header.number(), "found canonical state for block in database");
+            debug!(target: "engine::tree", ?hash, number = ?header.number(), "found canonical state for block in database");
```

Comment:
Why change to Debug? you should generally always prefer Display (%)

Reply:
whoops, artifact from when i removed these logs since they were weirdly showing up as taking a lot of time in some profiles